tests/tga.rs

```diff
@@ -0,0 +1,9 @@
+#[cfg(test)]
+use imagesize::size;
+
+#[test]
+fn exr_test() {
+    let dim = size("tests/images/tga/test.tga").unwrap();
```

Comment:
Didn't mention this in the larger TGA post but if we start checking files based on `image_type`, then we likely would want to source TGA files of varying types to verify they all work correctly.

Reply:
tga is sooooo tricky.... why they define a header like that...